tools/postinstall

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+pod_install() (
```

Comment:
Mmm, makes sense. I didn't know about `return`,  but I did find the round parens (and I think you've mentioned those before), so I was actually using them intentionally, so that the first exit (which applies unless you're on macOS) would only exit the function and not the whole script. But that second `exit`, if we don't have CocoaPods, *should* exit the whole script, which I guess I forgot about.

And it sounds like curly braces is the done thing, so I'm much happier to use them, now I know about `return`. 🙂

Reply:
Yep -- subshells are useful, but I would be super confused to see a codebase where they were used on all functions :slightly_smiling_face:  If you did want a subshell to apply to a whole function, I think I'd want to write the braces anyway and then the parens inside those, so it stood out as more visually distinct.